examples/gnrc_networking/Makefile

```diff
@@ -47,11 +47,11 @@ USE_ZEP ?= 0
 # set the ZEP port for native
 ZEP_PORT_BASE ?= 17754
 ifeq (1,$(USE_ZEP))
-  TERMFLAGS += -z [::1]:$(ZEP_PORT_BASE)
+  TERMFLAGS += --native-args '-z [::1]:$(ZEP_PORT_BASE)'
   USEMODULE += socket_zep
 
   ifneq (,$(ZEP_MAC))
-    TERMFLAGS += --eui64=$(ZEP_MAC)
+    TERMFLAGS += --native-args '\-\-eui64=$(ZEP_MAC)'
   endif
 endif
```

Comment:
If you are touching all the application makefiles anyway, you could also move this to a common place :innocent:

Reply:
I would be happy to comply but the actual added flags seem to be pretty diverse.

Reply:
I came up with a proposal for that. I also reverted to call native's binary directly for tests since pyterm isn't beneficial here and just adds overhead.